modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -56,21 +63,74 @@ You can configure PTP fast events for vRAN clusters that are deployed using the
+<1> Can be one of `PtpConfigMaster.yaml`, `PtpConfigSlave.yaml`, or `PtpConfigSlaveCvl.yaml` depending on your requirements. `PtpConfigSlaveCvl.yaml` configures `linuxptp` services for an Intel E810 Columbiaville NIC. For configurations based on `group-du-sno-ranGen.yaml` or `group-du-3node-ranGen.yaml`, use `PtpConfigSlave.yaml`.
 <2> Device specific interface name.
 <3> You must append the `--summary_interval -4` value to `ptp4lOpts` in `.spec.sourceFiles.spec.profile` to enable PTP fast events.
 <4> Required `phc2sysOpts` values. `-m` prints messages to `stdout`. The `linuxptp-daemon` `DaemonSet` parses the logs and generates Prometheus metrics.
 <5> Optional. If the `ptpClockThreshold` stanza is not present, default values are used for the `ptpClockThreshold` fields. The stanza shows default `ptpClockThreshold` values. The `ptpClockThreshold` values configure how long after the PTP master clock is disconnected before PTP events are triggered. `holdOverTimeout` is the time value in seconds before the PTP clock event state changes to `FREERUN` when the PTP master clock is disconnected. The `maxOffsetThreshold` and `minOffsetThreshold` settings configure offset values in nanoseconds that compare against the values for `CLOCK_REALTIME` (`phc2sys`) or master offset (`ptp4l`). When the `ptp4l` or `phc2sys` offset value is outside this range, the PTP clock state is set to `FREERUN`. When the offset value is within this range, the PTP clock state is set to `LOCKED`.
 
-. Apply the following `PolicyGenTemplate` changes to your specific site YAML files, for example, `example-sno-site.yaml`:
-
-.. In `.sourceFiles`, add the `Interconnect` CR file that configures the AMQ router to the `config-policy`:
+. Configure a `LocalVolume` (`LV`) to persist the PTP events subscription on the host.
++
+For example, add the following YAML to the `group-du-sno-ranGen.yaml` file:
 +
 [source,yaml]
 ----
-- fileName: AmqInstance.yaml
+- fileName: StorageLV.yaml <1>
   policyName: "config-policy"
+  spec:
+    storageClassDevices:
+      - storageClassName: "storage-class-http-events" <2>
+        volumeMode: Filesystem
+        fsType: xfs
+        devicePaths:
+          - /dev/disk/by-partlabel/httpevent1
+          - /dev/disk/by-partlabel/httpevent2
+- fileName: StoragePV.yaml  <3>
 ----
++
+<1> PTP events with HTTP transport requires local storage on the cluster node provided by the Local Storage Operator.
+<2> You need `storage-class-http-events` to use HTTP transport with the PTP Operator.
+Use the `ignitionConfigOverride` field in the related `SiteConfig` CR to manage the disk labels.
+<3> Optional. Configures storage for the `cloud-event-proxy` pubsubstore.
```

Comment:
<3> Optional. Configures storage for the `cloud-event-proxy` pubsubstore. This is only needed if PVs are not dynamically created by the local storage operator.